src/app/pages/admin/admin.controller.js

```diff
@@ -3,35 +3,52 @@
 export default class AdminController {
     constructor($rootScope, syncDataService, adminService, sharedAdminFactory) {
         'ngInject';
+
+        // this.listOfUsers = null;
         this.rootScope = $rootScope;
         this.syncDataService = syncDataService;
         this.adminService = adminService;
-        syncDataService.getAllUsersFromFirebase();
-
-        this.listOfUsers = this.rootScope.listOfUsers;
-        this.filteredItems = [];      
-        this.headers =  ['Id','Name','E-mail','Admin','Password','History',' Profile'];
-        this.sort = {       
-            sortingOrder : 'id',
-            reverse : false
+        const data = Object.values($rootScope.listOfUsers).filter(item => {
+            if (item && item.email) {
+                return item
+            }
+        });
+
+            this.listOfUsers = data.map(({ userId, firstName, lastName, email, phone }) => ({
```

Comment:
explain why do you need this line

Reply:
Because data has extra info which I don't need, that's why I just choose items what I'll be use in my table.